src/index.css

```diff
@@ -1,9 +1,6 @@
-@tailwind base;
-@tailwind components;
-@tailwind utilities;
+@import 'tailwindcss/base';
 
-@layer components {
-  .ock-label {
-    @apply text-base font-medium leading-4 text-black;
-  }
-}
+@import 'tailwindcss/components';
+@import './token/components/TokenChip.css';
+
+@import 'tailwindcss/utilities';
```

Comment:
I don't think we should include `components` and `utility`, maybe not even `base`.

If we want to avoid to have OnchainKit css override other apps css.

Reply:
removed